### PR TITLE
12: Time entries content type.

### PR DESCRIPTION
**Pull request by josue2591**
**Steps to test**
- Check if the date dependency was added automatically.
- Verify the manatime_feature_time_entries was created fine.
- Check if the human and machine names are right.
- Check the Time entries's fields.
- Review if manatime_feature_time_entries was added fine to settings.php.

[Issue link](https://github.com/ManatiCR/manatime/issues/12)

### CODE REVIEW

File: modules/features/manatime_feature_time_entries/manatime_feature_time_entries.features.field_base.inc [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        ),
+      ),
+      'target_type' => 'node',
+    ),
+    'translatable' => 0,
+    'type' => 'entityreference',
+  );
+
+  return $field_bases;
+}
```
Comment (rigoucr) [inline modules/features/manatime_feature_time_entries/manatime_feature_time_entries.features.field_base.inc]: All this field bases should be in to the **manatime_feature_field_bases** feature,  not here. :ghost:

**Comment by josue2591** [comment]
Removed field bases from manatime_feature_time_entries and added to manatime_feature_field_bases.

File: modules/features/manatime_feature_time_entries/manatime_feature_time_entries.info [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+features[field_instance][] = node-time_entries-field_status
+features[field_instance][] = node-time_entries-field_task
+features[node][] = time_entries
+features[variable][] = field_bundle_settings_node__time_entries
+features[variable][] = menu_options_time_entries
+features[variable][] = menu_parent_time_entries
+features[variable][] = node_options_time_entries
+features[variable][] = node_preview_time_entries
+features[variable][] = node_submitted_time_entries
+features_exclude[field_base][body] = body
```
Comment (kporras07) [inline modules/features/manatime_feature_time_entries/manatime_feature_time_entries.info]: you can remove these 5 lines related to field_base.

File: modules/features/manatime_feature_time_entries/manatime_feature_time_entries.features.field_base.inc [hunk 3]
```diff
@@ -0,0 +1,122 @@
+<?php
```
Comment (kporras07) [inline modules/features/manatime_feature_time_entries/manatime_feature_time_entries.features.field_base.inc]: This file is now unused, please remove it.

**Comment by josue2591** [comment]
Deleted unnecessary file and deleted lines from manatime_feature_time_entries.info.

File: modules/features/manatime_feature_field_bases/manatime_feature_field_bases.features.field_base.inc [hunk 4]
```diff
@@ -81,13 +81,77 @@ function manatime_feature_field_bases_field_default_field_bases() {
...
+      ),
+      'target_type' => 'node',
+    ),
+    'translatable' => 0,
+    'type' => 'entityreference',
+  );
+
+  // Exported field_base: 'field_date'.
+  $field_bases['field_date'] = array(
+    'active' => 1,
```
Comment (kporras07) [inline modules/features/manatime_feature_field_bases/manatime_feature_field_bases.features.field_base.inc:121]: We need date to only contain: day, month and year. Besides that, we also need to collect end date in the same field.

**Comment by kporras07** [comment]
I'd make status look like this: 
screenshot from 2016-09-06 17 39 59

**Comment by josue2591** [comment]
Changes applyed to the fields date and status.

**Pull request merged**